server.py

```diff
@@ -71,7 +70,15 @@ def post_collect():
     next_service = APP.config['NEXT_MICROSERVICE_HOST']
     source_id = input_data.get('payload_id')
 
-    workers.download_job(input_data['url'], source_id, next_service)
+    b64_identity = request.headers['x-rh-identity'] \
+        if request.headers.get('x-rh-identity') else None
```

Comment:
Note that this usage takes care of `KeyError`

Reply:
You can do the same job by a simple:

```suggestion
   b64_identity = request.headers.get('x-rh-identity')
```

Because that's exactly the job of [`dict.get`](https://docs.python.org/3/library/stdtypes.html#dict.get) to handle the `KeyError`.